File: src/main/resources/output/IFC4X3_java/Entities/IfcRepresentationItem/IfcGeometricRepresentationItem/IfcGeometricSet/IfcGeometricSet.java
public class IfcGeometricSet extends IfcGeometricRepresentationItem {
    public SET [1:?] OF IfcGeometricSetSelect Elements;

    // === EXTENDED BY ===
    // IfcGeometricCurveSet

    // === WHERE CLAUSES ===
    // ConsistentDim : SIZEOF(QUERY(Temp <* Elements | Temp.Dim <> Elements[1].Dim)) = 0

    // === DERIVE CLAUSES ===
    // Dim : IfcDimensionCount := Elements[1].Dim
}

File: src/main/resources/output/IFC4X3_vbnet/Entities/IfcRepresentationItem/IfcGeometricRepresentationItem/IfcGeometricSet/IfcGeometricSet.vb
Public Class IfcGeometricSet Inherits IfcGeometricRepresentationItem
    Public Elements As SET [1:?] OF IfcGeometricSetSelect

    ' === EXTENDED BY ===
    ' IfcGeometricCurveSet

    ' === WHERE CLAUSES ===
    ' ConsistentDim : SIZEOF(QUERY(Temp <* Elements | Temp.Dim <> Elements[1].Dim)) = 0

    ' === DERIVE CLAUSES ===
    ' Dim : IfcDimensionCount := Elements[1].Dim
End Class
